src/Language/PureScript/TypeChecker/Entailment.hs

```diff
@@ -103,6 +105,15 @@ data SolverOptions = SolverOptions
   -- ^ Should the solver be allowed to defer errors by skipping constraints?
   }
 
+-- | Build a type class instance dictionary for the `IsSymbol` class for a symbol
+isSymbolDictionary :: String -> TypeClassDict
```

Comment:
I originally read this as a question: "is this a symbol dictionary?" 😄 

Maybe something like `makeIsSymbolDictionary` would be better?